server/lib/template/helpers/finance.js

```diff
@@ -20,7 +20,15 @@ const EUR_FMT = {
   format : '%v&nbsp;%s', // value before symbol
 };
 
-const FORMATS = [null, FC_FMT, USD_FMT, EUR_FMT]; // NB: Indeces must match currency IDs in database
+const ROUBLE_FMT = {
```

Comment:
Looks like this exchange rate testing code accidently got incorporated.  The PR should be updated to remove this.